reflex/components/component.py

```diff
@@ -446,25 +446,24 @@ def __str__(self) -> str:
 
         return _compile_component(self)
 
-    def _apply_theme(self, theme: Component):
+    def _apply_theme(self, theme: Optional[Component]):
         """Apply the theme to this component.
 
         Args:
             theme: The theme to apply.
         """
         pass
 
-    def apply_theme(self, theme: Component):
+    def apply_theme(self, theme: Optional[Component]):
```

Comment:
Maybe just do the `None` check here and don't recurse if it's none. Alternatively, can we keep it where it was above?

Reply:
This PR need it because apply_theme was never called at all when theme=None otherwise.